src/js/actions/search/libraries.js

```diff
@@ -91,7 +91,7 @@ define(function (require, exports) {
                         name: title,
                         pathInfo: library.name + ": " + strings.SEARCH.CATEGORIES[categoryKey],
                         iconID: _getSVGClass([categoryKey]),
-                        category: ["LIBRARY", library.name, categoryKey]
+                        category: ["LIBRARY", library.id.replace(/-/g, "."), categoryKey]
```

Comment:
@iwehrman Yes, the search store makes every item ID its broad category and some unique ID, separated by a hyphen. Then later the search store splits the ID on the hyphen. This should only really be an issue when a category is a user inputted string, because otherwise we are using the keys to localized strings.

The reason there's been a tendency to store information in the ID is because the whole Datalist/Select structure only passes around IDs when something is selected or confirmed, so I've been trying not to overhaul that.